dist/edu-benefits-schema.json

```diff
@@ -662,7 +606,7 @@
             "$ref": "#/definitions/dateRange"
           },
           "hours": {
-            "type": "integer"
+            "type": "number"
```

Comment:
Hmm, maybe I misunderstood. I thought the issue was that we wanted them to match and for the front end to not send NaN or null.

We can definitely change it, though do we want to be more restrictive? I think decimals could be valid for coursework hours.